### PR TITLE
Update ProjectCharter.md - Adjust license rule for Marketing Working Group

### PR DESCRIPTION
**Pull request by MarkusKuemmerle**

### CODE REVIEW

File: ProjectCharter.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ Project participants acknowledge that the copyright in all new contributions wil
...
-- Only exception is the documentation contributed to or created by the Outreach Committee (former Marketing Working Group) in the folder [https://github.com/camaraproject/WorkingGroups/tree/main/Marketing](https://github.com/camaraproject/WorkingGroups/tree/main/Marketing). For that the [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/legalcode.txt) shall apply. It is strongly forbidden to save any API-related definition, documentation or code in this folder.
+- Only exception is the documentation contributed to or created by the Outreach Committee (former Marketing Working Group) in the reoository [https://github.com/camaraproject/Marketing](https://github.com/camaraproject/Marketing). For that the [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/legalcode.txt) shall apply. It is strongly forbidden to save any API-related definition, documentation or code in this repository.
```
Comment (hdamker) [inline ProjectCharter.md:251]:
```suggestion
- Only exception is the documentation contributed to or created by the Outreach Committee (former Marketing Working Group) in the repository [https://github.com/camaraproject/Marketing](https://github.com/camaraproject/Marketing). For that the [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/legalcode.txt) shall apply. It is strongly forbidden to save any API-related definition, documentation or code in this repository.
```

**Review by hdamker — CHANGES_REQUESTED** [review]
LGTM, one typo

**Comment by hdamker** [comment]
@MarkusKuemmerle Please add a description of the PR and a link to the issue. Otherwise it is difficult to understand later why this PR happened.

**Comment by MarkusKuemmerle** [comment]
Working Groups are seperated in different repos. Links have to be adjusted.
See https://github.com/camaraproject/Governance/issues/84.

**Review by hdamker — APPROVED** [review]

**Pull request merged**